index.js

```diff
@@ -55,7 +55,8 @@ var defaultPluralRules = {
       if (lastTwo >= 3 && lastTwo <= 10) return 3;
       return lastTwo >= 11 ? 4 : 5;
     },
-    bosnian_serbian: russianPluralGroups,
+    serbian: russianPluralGroups,
+    bosnian: russianPluralGroups,
```

Comment:
`bosnian_serbian` must be preserved, or else it's a breaking change.
```suggestion
    bosnian_serbian: russianPluralGroups,
    serbian: russianPluralGroups,
    bosnian: russianPluralGroups,
```